api/renter_test.go

```diff
@@ -66,4 +67,5 @@ func TestUploadAndDownload(t *testing.T) {
 	if upRoot != downRoot {
 		t.Error("uploaded and downloaded file have a hash mismatch")
 	}
+	os.Remove(downloadName)
```

Comment:
good catch, for some reason it didn't occur to me to move the downloaded file to the tempdir.